extensions/amp-accordion/amp-accordion.md

```diff
@@ -26,9 +26,11 @@ limitations under the License.
 
 # amp-accordion
 
-Provides a way for viewers to glance at the content outline and jump to any section. This is helpful for mobile devices where even a couple of sentences into a section requires scrolling.
-
 <table>
+  <tr>
+    <td class="col-fourty"><strong>Description</strong></td>
+    <td>A clickable stacked list of headers that reveal or hide content associated with them.</td>
```

Comment:
While technically correct, I find this a bit hard to understand. Maybe use the first sentence instead: `The `amp-accordion` component allows you to display collapsible and expandable content sections.`

Reply:
I don't think thats quite right either. I've updated it to something in the middle.